src/generated/cloud/compute/v1/src/model.rs

```diff
@@ -646,6 +723,478 @@ impl wkt::message::Message for MachineTypeAggregatedList {
     }
 }
 
+/// Defines additional types related to [MachineTypeAggregatedList].
+pub mod machine_type_aggregated_list {
+    #[allow(unused_imports)]
+    use super::*;
+
+    /// The message type for the [warning][google.cloud.compute.v1.MachineTypeAggregatedList.warning] field.
+    ///
+    /// [google.cloud.compute.v1.MachineTypeAggregatedList.warning]: crate::model::machine_type_aggregated_list::Warning
+    #[derive(Clone, Default, PartialEq)]
+    #[non_exhaustive]
+    pub struct Warning {
```

Comment:
This message type is duplicated (with slight variations in the comments) 184 times in the compute spec. Yuck.